src/app/app.component.html

```diff
@@ -1 +1,14 @@
-<div>App is working</div>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <app-chatroom></app-chatroom>
```

Comment:
Use router outlet and Angular's routing, so we can navigate to other screens too.